app/models/review_request.rb

```diff
@@ -0,0 +1,5 @@
+class ReviewRequest < ApplicationRecord
+  belongs_to :owner, class_name: "User", foreign_key: "owner_id", inverse_of: :owned_review_requests
+  belongs_to :pull_request, inverse_of: :review_requests
+  has_and_belongs_to_many :users
```

Comment:
A personal opinion, I'd always go with `has_many :through` because it's more versatile

Reply:
I agree with Santi. I would use `has_many :through` here and below.

Reply:
I had initially thought that a review request is related to multiple users due to the array requested_reviewers in the response, but it turns out it sends a unique requested_reviewer per request. I think it's better to relate it to a single user and a pull request, to simulate the request better, and remove the habtm relation entirely. What do you think?

Reply:
makes sense